Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -28,9 +28,11 @@ import Workspace
 import Dispatch
 import struct Foundation.URL
 
+import struct Basics.AbsolutePath
+import struct Basics.TSCAbsolutePath
```

Comment:
What’s the rule about using `TSCAbsolutePath` vs. `AbsolutePath`? Or which features will one support but not the other?

Reply:
we are slowly transitioning SwiftPM code base off of TSC. I wanted to keep the change set smaller so lets all the LSP code using the TSC version, but since some of SwiftPM public APIs are now using SwiftPM's AbsolutePath we need do some bridging in SwiftPMWorkspace. eventually all of SwiftPM public APIs will be based on swift-system Path so we will be aligned again, this is the first step to get there